packages/api/wrangler.toml

```diff
@@ -21,12 +21,17 @@ NFT_STORAGE_API = "https://api.nft.storage"
 DEBUG = "false"
 ENV = "production"
 GATEWAY_DOMAIN = "nftstorage.link"
-IPFS_GATEWAYS = "[\"https://ipfs.io\", \"https://cf.nftstorage.link\", \"https://nft-storage.mypinata.cloud/\"]"
 
 [[env.production.r2_buckets]]
 bucket_name = "super-hot"
 binding = "SUPERHOT"
 
+[[env.production.unsafe.bindings]]
```

Comment:
are `unsafe.bindings` documented anywhere? Can you link to docs or add a note here about how they work and why they may or may not be unsafe?

Reply:
All i can find is this PR which mentions they are undocumented https://github.com/cloudflare/wrangler2/issues/1004

Reply:
and this... the list of drawbacks listed on this PR is scary
https://github.com/cloudflare/wrangler2/pull/906

Reply:
The unsafe is not needed anymore in fact. They already support without it :)

The drawbacks there seem outdated by know from what I have seen in their discord